aule/resources/idl/alfa.idl

```diff
@@ -19,46 +19,28 @@ interface NameSpaceDeclaration <: Statement {
 interface PolicySetDeclaration <: Statement {
     name: str;
     algorithm: ApplyStatement;
-    targetStatement: TargetStatement;
-    conditionStatement: ConditionStatement;
+    target: TargetStatement;
     references: [Identifier];
     policies: [PolicyDeclaration];
     policysets: [PolicySetDeclaration];
-    events: [EventDeclaration];
     modifiers: [Keywords];
```

Comment:
We do not have a node called `Keyword`(s). Perhaps we should add it as a parent of `ExportKeyword`

Reply:
Done